content/docs/tasks/security/rbac-groups/index.md

```diff
@@ -0,0 +1,303 @@
+    kind: "Policy"
+    metadata:
+      name: "require-mtls-jwt"
+    spec:
+      targets:
+      - name: httpbin
+      peers:
+      - mtls: {}
+      origins:
+      - jwt:
+          issuer: "testing@secure.istio.io"
+          jwksUri: "{{< github_file >}}/security/tools/jwt/samples/jwks.json"
+      principalBinding: USE_ORIGIN
+    EOF
+    {{< /text >}}
+
+1.  Set the environmental variable TOKEN to contain a valid sample JWT.
+This tutorial uses
+[this sample JWT]({{< github_file >}}/security/tools/jwt/samples/groups-scope.jwt).
+The sample JWT contains a JWT claim with a `groups` claim key and a list of
+strings, [`"group1"`, `"group2"`] as the claim value.
+The JWT claim value could
+either be a string or a list of strings; both types are supported.
+
+    {{< text bash>}}
+    $ TOKEN=$(curl {{< github_file >}}/security/tools/jwt/samples/groups-scope.jwt -s)
+    {{< /text >}}
+
+1.  Connect to the httpbin service:
+
+    {{< text bash >}}
+    $ kubectl exec $(kubectl get pod -l app=sleep -n $NS -o jsonpath={.items..metadata.name}) -c sleep -n $NS -- curl http://httpbin.$NS:8000/ip -s -o /dev/null -w "%{http_code}\n" --header "Authorization: Bearer $TOKEN"
+    {{< /text >}}
+
+    When a valid JWT is attached, it should return HTTP code 200:
+
+    {{< text plain >}}
+    200
+    {{< /text >}}
+
+1.  Verify that the connection to the `httpbin` service fails when the JWT is not attached:
+
+    {{< text bash >}}
+    $ kubectl exec $(kubectl get pod -l app=sleep -n $NS -o jsonpath={.items..metadata.name}) -c sleep -n $NS -- curl http://httpbin.$NS:8000/ip -s -o /dev/null -w "%{http_code}\n"
+    {{< /text >}}
+
+    When no valid JWT is attached, it should return HTTP code 401:
+
+    {{< text plain >}}
+    401
+    {{< /text >}}
+
+## Configure groups-based authorization
+
+This section creates a policy to authorize the access to the `httpbin`
+service if the requests are originated from specific groups.
+As there may be some delays due to caching and other propagation overhead,
+wait a moment for the newly defined RBAC policy to become effective.
+
+1.  Enable the Istio RBAC for the namespace:
+
+    {{< text bash >}}
+    $ cat <<EOF | kubectl apply -n $NS -f -
+    apiVersion: "rbac.istio.io/v1alpha1"
+    kind: RbacConfig
+    metadata:
+      name: default
+    spec:
+      mode: 'ON_WITH_INCLUSION'
+      inclusion:
+        namespaces: ["rbac-groups-test-ns"]
+    EOF
+    {{< /text >}}
+
+1.  Run the following command to verify that the curl connection
+to the httpbin service is rejected after the RBAC policy has become effective.
+
+    {{< text bash >}}
+    $ kubectl exec $(kubectl get pod -l app=sleep -n $NS -o jsonpath={.items..metadata.name}) -c sleep -n $NS -- curl http://httpbin.$NS:8000/ip -s -o /dev/null -w "%{http_code}\n" --header "Authorization: Bearer $TOKEN"
+    {{< /text >}}
+
+    When the RBAC policy has become effective, it should return HTTP code 403:
```

Comment:
Word choice problems:
Once the RBAC policy takes effect, the command returns the HTTP code 403.
Please make the changes in word choice for all instances.

Reply:
Done.